lib/sign_in/logingov/well_known/service.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module SignIn
+  module WellKnown
+    class Service
```

Comment:
This class doesn't follow the structure of a Ruby class nor a service object. https://dsva.slack.com/archives/C04LW2P19M0/p1682543003130879?thread_ts=1682533636.044009&cid=C04LW2P19M0